source/elements/oneDAL/source/algorithms/clustering/kmeans.rst

```diff
@@ -9,6 +9,33 @@ The K-Means algorithm partitions :math:`n` feature vectors into :math:`k`
 *clusters* minimizing some criterion. Each cluster is characterized by a
```

Comment:
Don't you want to connect "clusters" with "Clustering" task in the glossary or create its own glossary term?